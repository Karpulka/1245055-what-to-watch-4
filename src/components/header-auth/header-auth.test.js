import React from "react";
import renderer from "react-test-renderer";
import HeaderAuth from "./header-auth";

it(`Render HeaderAuth`, () => {
  const tree = renderer
    .create(<HeaderAuth />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
