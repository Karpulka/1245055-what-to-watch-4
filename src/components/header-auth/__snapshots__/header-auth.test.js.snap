// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HeaderAuth 1`] = `
<h1
  className="page-title user-page__title"
>
  Sign in
</h1>
`;
